System-Testing-Deployments-and-Upgrades/Control-System-Upgrade/Getting-Ready/index.rst

```diff
@@ -3,64 +3,64 @@
-#. Go over the work in the `appropriate release <https://jira.lsstcorp.org/projects/CAP?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=unreleased>`_ in the CAP Jira project.
-    * This is identified by label of: ``ts_xml X.Y``.
-    * Use the ``release_tickets`` script in the vanward_ package.
-#. Ensure that all work merged in the `ts_xml <https://github.com/lsst-ts/ts_xml.git>`_ repository has a ticket associated with it in the release.
+#. Coordinate with major stakeholders at the Commissioning Activities Planning (CAP) meeting to discuss the upcoming release.
+#. Determine the summit deployment date and work backwards to set the XML work closure deadline.
+    * Consider observing runs and special overnight work at the observatory.
+    * Typically allow about two weeks between work closure and deployment.
+    * This period may be extended if extra testing or other circumstances require it.
+    * See :ref:`Control-System-Upgrade-Getting-Ready-Setting-Schedule` for further instructions.
+    * Once the deployment date is set, run the ``create_summit_upgrade_ticket`` script in the vanward_ package to create the appropriate Jira ticket.
+#. Create a cycle build Confluence page (`Software Upgrades <https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=LSSTCOM&title=Software+Upgrades>`_) listing core package versions and any operational changes.
+#. Present the schedule at the CAP meeting at least one week before the work closure deadline.
+#. Review the release in the `CAP Jira project <https://rubinobs.atlassian.net/projects/CAP?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page>`_, identified by the label ``ts_xml X.Y``.
+    * Use the ``release_tickets`` script from the vanward_  package.
+#. Confirm all merged work in the `ts_xml <https://github.com/lsst-ts/ts_xml.git>`_ repository is linked to a Jira ticket in the release.
     * Folks should be following the `XML Work Reporting <https://tssw-developer.lsst.io/development-guidelines/xml/reporting-xml-release-work.html#reporting-xml-release-work>`_ procedure.
     * Use the ``find_merges_without_release_tickets`` script in the vanward_ package.
-#. Send a reminder about the work closure deadline at least one day prior AND the day of the closure.
-#. Ensure that all work tickets are closed when the deadline passes (use Step 4 script).
+#. Send reminders about the work closure deadline at least one day prior AND on the day of the deadline.
+#. Ensure all work tickets are closed when the deadline passes (use the Step 5 script).
 #. Work with the Telescope and Site Build Engineer on the day of the artifact build to go over any potentially open work and sign off on all software versions being used.
+#. For **incremental upgrades**, use the ``move_bucket_ticket_links`` script in the vanward_ package to transfer changes not included in the mini update to the next full release cycle.
```

Comment:
This step is out of place with the others and is entirely related to the incremental upgrade process. I would create a separate sub-section here and start outlining what we do know about this process.